src/class-fields/initializer-eval-super/cls-decl-fields-indirect-eval.template

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-performeval-rules-in-initializer
+path: language/statements/class/fields-derived-cls-indirect-
+name: indirect eval
+---*/
+
+class A = {}
+
+var executed = false;
+class C extends A = {
+  x = (0, eval)('executed = true; /*{ initializer }*/;';
+}
+
+assert.throws(/*{ execution error }*/, function() {
```

Comment:
the reutilization of the templates using execution and early errors fields is great! +1

Reply:
the templates still remain with the "spaced" names.